content/news/2023-03-31-release-23.03-highlights.md

```diff
@@ -0,0 +1,123 @@
++++
+title = "Release 23.03 Highlights"
+date = 2023-03-31T00:00:00Z
+type = "post"
+description = "Highlights of the 23.03 release."
+in_search_index = true
++++
+
+Helix is a modal text editor with built-in support for multiple selections,
+Language Server Protocol (LSP), tree-sitter, and experimental support for Debug
+Adapter Protocol (DAP).
+
+Today marks the 23.03 release. First, a big _thank you_ to everyone involved!
+This release includes changes from 102 contributors. Today's release brings
+some especially useful and exciting features, so let's jump right in.
+
+## Soft-wrap
+
+{{ asciinema(id="softwrap", width="94", height="25") }}
+
+Soft-wrap breaks up lines to fit in view without inserting actual line break
+characters into the text (so called "hard-wrapping"). This can be useful for
+reading and editing documentation or code with especially long lines. Enable
+soft-wrap in your config with the `editor.soft-wrap.enable` key:
+
+```toml
+# ~/.config/helix/config.toml
+[editor.soft-wrap]
+enable = true
+```
+
+Heuristics for where to break lines and the wrap indicator may also be
+[configured](https://docs.helix-editor.com/configuration.html#editorsoft-wrap-section).
+
+Soft-wrap is the first feature implemented on top of a new "virtual text" API
+within the Helix codebase. The virtual text API allows us to decorate and
+manipulate the actual text in a document without disturbing positioning and
+modifications. The virtual text API opens the door for many new exciting
+features like the inlay hints we'll see next.
+
+## Inlay hints
+
+{{ asciinema(id="inlay-hints", width="94", height="25") }}
+
+Inlay hints are a new feature in the latest LSP specification. Language servers
+can provide editors with hints to show in a document. In the example above,
+rust-analyzer provides type hints for variables and intermediary expressions
+as well as parameter names in function calls.
+
+Inlay hints can be enabled in your config with the `lsp.display-inlay-hints`
+key.
+
+## Initial support for snippets
+
+{{ asciinema(id="snippets", width="94", height="25") }}
+
+LSP snippets are a way for language servers to provide more complex completions
+for an editor. Language servers can specify where to place cursors within a
+completion (tabstops), placeholder values and more. Helix now has initial
+support for LSP completions, leaving more advanced snippet features like
+virtual placeholders, multiple tabstops, and variable substitutions for future
+work.
```

Comment:
One thing I think would be useful to put here is how helix control jumping between insert points in snippet, maybe we want to answer that here? Usually when I use snippet, I have no idea how.

Reply:
Currently that isn't supported. We only automatically jump to the first tabstop and there isn't a way to jump between tabstops